### PR TITLE
fix: display active indicator when tab is disabled

### PR DESCRIPTION
**Pull request by m4thieulavoie**
# Description

When setting the _last_ tab to `disabled`, the active indicator would not show up at all. This PR is to fix that

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-foundation/src/tabs/tabs.ts
Comment (m4thieulavoie) [inline packages/web-components/fast-foundation/src/tabs/tabs.ts:191]: With the change, the active indicator will now show up as soon as at least _one_ tab is focusable

Comment (scomea) [reply packages/web-components/fast-foundation/src/tabs/tabs.ts:191]: The query seems unnecessary since we are already iterating through the tabs in the forEach loop immediately beforehand.  Could we not just set ` this.showActiveIndicator = false;` before the forEach loop and remove the trailing else that sets it to false as you have already done?

Comment (m4thieulavoie) [reply packages/web-components/fast-foundation/src/tabs/tabs.ts]: Indeed, applying the fix right now!

**Review by scomea — APPROVED** [review]

**Review by EisenbergEffect — APPROVED** [review]

**Comment by EisenbergEffect** [comment]
Ready to merge pending final approval from @chrisdholt

File: packages/web-components/fast-foundation/src/tabs/tabs.spec.ts
Comment (SethDonohue) [inline packages/web-components/fast-foundation/src/tabs/tabs.spec.ts:359]: nit:
```suggestion
        it("should not display an active indicator if all tabs are disabled", async () => {
```

**Review by SethDonohue — APPROVED** [review]

File: packages/web-components/fast-foundation/src/tabs/tabs.spec.ts
Comment (m4thieulavoie) [reply packages/web-components/fast-foundation/src/tabs/tabs.spec.ts]: Thanks!

**Comment by SethDonohue** [comment]
Just a note, this also seems to fix an issue that I just noticed where setting `activeindicator=false` on tabs does nothing. This 
 change corrects that as well.
 
 Before:
Screen Shot 2020-12-28 at 3 28 11 PM

After:
Screen Shot 2020-12-28 at 3 27 49 PM

**Review by nicholasrice — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]
Thanks @@m4thieulavoie!

**Pull request merged**
